app/src/main/java/openfoodfacts/github/scrachx/openfood/utils/CoroutineDispatchers.kt

```diff
@@ -0,0 +1,21 @@
+package openfoodfacts.github.scrachx.openfood.utils
+
+import kotlinx.coroutines.CoroutineDispatcher
+import kotlinx.coroutines.Dispatchers
+import javax.inject.Inject
+
+interface CoroutineDispatchers {
+
+    fun main(): CoroutineDispatcher
+    fun io(): CoroutineDispatcher
+    fun default(): CoroutineDispatcher
+}
+
+class CoroutineDispatchersImpl @Inject constructor() : CoroutineDispatchers {
```

Comment:
Why are we using this? Why don't we use default dispatchers?

Reply:
for mock default dispatchers in test, runTestBlocking didn't work for me, maybe because we have rxjava2->coroutine mapper inside